### PR TITLE
[Backend] Added endpoint for checking if named graph exists in Virtuoso

### PR DESCRIPTION
**Pull request by Ponsietta**

### CODE REVIEW

File: src/backend/src/main/java/com/linkedpipes/lpa/backend/controllers/VirtuosoController.java [hunk 1]
```diff
@@ -21,4 +32,13 @@
     public ResponseEntity<String> serviceDescription(@RequestParam(value = "graphId") String graphId) {
         return ResponseEntity.ok(TtlGenerator.getVirtuosoServiceDescription(GRAPH_NAME_PREFIX + graphId));
     }
+
+    @GetMapping("api/virtuoso/graphExists")
+    public ResponseEntity checkNamedGraphExists(@RequestParam(value = "graphId") String graphId) throws LpAppsException {
+        boolean exists = virtuosoService.checkNamedGraphExists(graphId);
```
Comment (eghuro) [inline src/backend/src/main/java/com/linkedpipes/lpa/backend/controllers/VirtuosoController.java]: Shouldn't it be `GRAPH_NAME_PREFIX + graphId` here?

**Comment by aorumbayev** [comment]
@Ponsietta also please confirm that it works by testing it manually via postman

File: src/backend/src/main/java/com/linkedpipes/lpa/backend/controllers/VirtuosoController.java (hunk 1 above)
Comment (Ponsietta) [reply src/backend/src/main/java/com/linkedpipes/lpa/backend/controllers/VirtuosoController.java]: Doesn't apply to this case

**Comment by Ponsietta** [comment]
> @Ponsietta also please confirm that it works by testing it manually via postman

@aorumbayev I did and it works fine

**Comment by Ponsietta** [comment]
@aorumbayev tests are running fine now. Will you be adding usage of the endpoint on this branch before closing pull request?

**Comment by aorumbayev** [comment]
@Ponsietta yes ill approve and merge after ill add the frontend code

**Pull request merged**